src/main/java/racingcar/util/style/DistanceStyleFactory.java

```diff
@@ -0,0 +1,20 @@
+package racingcar.util.style;
+
+import racingcar.exception.ExceptionMessages;
+import racingcar.exception.RacingCarException;
+
+public class DistanceStyleFactory {
```

Comment:
Factory 클래스는 인스턴스를 만들 필요가 없으므로 아래와 같이 제한했으면 좋았을 것 같다.

```suggestion
public class DistanceStyleFactory {

    private DistanceStyleFactory() {
        // Empty
    }
```